src/Service.Host/package.json

```diff
@@ -30,75 +30,77 @@
-    "eslint-config-prettier": "^6.6.0",
-    "eslint-loader": "^3.0.2",
-    "eslint-plugin-chai-expect": "^2.0.1",
-    "eslint-plugin-import": "^2.18.2",
-    "eslint-plugin-jest": "^23.0.4",
-    "eslint-plugin-jsx-a11y": "^6.2.3",
-    "eslint-plugin-prettier": "^3.1.1",
-    "eslint-plugin-react": "^7.16.0",
-    "eslint-plugin-react-hooks": "^2.3.0",
-    "file-loader": "^4.2.0",
-    "jest": "^24.9.0",
-    "node-sass": "^4.13.0",
-    "postcss-loader": "^3.0.0",
-    "prettier": "^1.19.1",
+    "enzyme": "^3.11.0",
+    "enzyme-adapter-react-16": "^1.15.6",
+    "eslint": "^7.22.0",
+    "eslint-config-airbnb": "^18.2.1",
+    "eslint-config-prettier": "^8.1.0",
+    "eslint-loader": "^4.0.2",
+    "eslint-plugin-chai-expect": "^2.2.0",
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-jest": "^24.3.2",
+    "eslint-plugin-jsx-a11y": "^6.4.1",
+    "eslint-plugin-prettier": "^3.3.1",
+    "eslint-plugin-react": "^7.23.1",
+    "eslint-plugin-react-hooks": "^4.2.0",
+    "file-loader": "^6.2.0",
+    "jest": "^26.6.3",
+    "node-sass": "^5.0.0",
+    "postcss-loader": "^5.2.0",
+    "prettier": "^2.2.1",
     "raf": "^3.4.1",
-    "sass-loader": "^8.0.0",
-    "style-loader": "^1.0.0",
-    "url-loader": "^0.6.2",
-    "webpack": "^4.41.2",
-    "webpack-cli": "^3.3.10",
-    "webpack-dev-server": "^3.9.0"
+    "sass-loader": "^11.0.1",
+    "style-loader": "^2.0.0",
+    "url-loader": "^4.1.1",
+    "webpack": "^5.28.0",
+    "webpack-cli": "^4.5.0",
+    "webpack-dev-server": "^3.11.2"
   },
   "dependencies": {
-    "@date-io/moment": "^1.3.11",
-    "@linn-it/linn-form-components-library": "^10.12.5",
-    "@material-ui/core": "^4.6.1",
-    "@material-ui/icons": "^4.5.1",
-    "@material-ui/pickers": "^3.2.8",
-    "@material-ui/styles": "^4.6.0",
-    "@testing-library/jest-dom": "^4.2.4",
-    "core-js": "^3.4.1",
+    "@date-io/moment": "^2.10.8",
+    "@linn-it/linn-form-components-library": "^11.4.1",
+    "@material-ui/core": "^4.11.3",
+    "@material-ui/data-grid": "^4.0.0-alpha.23",
+    "@material-ui/icons": "^4.11.2",
+    "@material-ui/pickers": "^3.3.10",
+    "@material-ui/styles": "^4.11.3",
+    "@testing-library/jest-dom": "^5.11.10",
+    "core-js": "^3.9.1",
+    "decimal.js": "^10.2.1",
     "history": "^4.10.1",
-    "isomorphic-fetch": "^2.2.1",
-    "moment": "^2.24.0",
-    "moment-locales-webpack-plugin": "^1.1.2",
-    "notistack": "^0.9.6",
-    "oidc-client": "^1.9.1",
+    "isomorphic-fetch": "^3.0.0",
+    "moment": "^2.29.1",
+    "moment-locales-webpack-plugin": "^1.2.0",
+    "notistack": "^1.0.5",
+    "oidc-client": "^1.11.5",
     "prop-types": "^15.7.2",
-    "react": "^16.12.0",
-    "react-dom": "^16.12.0",
-    "react-hot-loader": "^4.12.18",
-    "react-redux": "^7.1.3",
-    "react-router": "^5.1.2",
-    "react-router-dom": "^5.1.2",
-    "redux": "^4.0.4",
-    "redux-api-middleware": "^3.0.1",
-    "redux-devtools": "^3.5.0",
+    "react": "^17.0.2",
```

Comment:
that's great you've tested all these upgrades. I'd started looking at them for stores but you saved me a job.

Reply:
I'm a bit suspicious about how painless it was... Might be a bit more to it in more complicated projects